src/translations/ca.translation

```diff
@@ -828,4 +888,48 @@
         <label>Configuració de EDA</label>
         <name>HEDA_Settings</name>
     </customTabs>
+    <quickActions>
```

Comment:
@skosgi1 I don't think we want the quickActions in all of the translation files. Can you remove these?

I'm going to go ahead and move this over to QA. Nice work on this PR!!!